spiffworkflow-backend/Dockerfile

```
@@ -22,31 +30,34 @@ FROM base AS deployment
 # Avoid host:container file ownership issues
 RUN git config --global --add safe.directory '*'
 
-
 ######################## - SETUP
 
 # Setup image for installing Python dependencies.
 FROM base AS setup
 
-RUN pip install uv==0.10.0
+# Astral is the company behind uv
+COPY --from=ghcr.io/astral-sh/uv:alpine /usr/local/bin/uv /usr/local/bin/uvx /usr/local/bin
```

Hadolint (2.14.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)